src/main/thrift/parquet.thrift

```diff
@@ -369,6 +369,12 @@ enum PageType {
   DATA_PAGE_V2 = 3;
 }
 
+enum SortOrder {
```

Comment:
Could this enum have a more specific name? Maybe `BoundaryOrder`?

Reply:
Done.